html/datalist/demos/browsers.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Пример списка опций — <datalist> — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    * {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+      font-family: "Roboto", sans-serif;
+    }
+  </style>
+</head>
+<body>
+  <label for="myBrowser">Выберите браузер из списка:</label>
+  <input list="browsers" id="myBrowser" name="myBrowser" />
```

Comment:
```suggestion
  <label for="my-browser">Выберите браузер из списка:</label>
  <input list="browsers" id="my-browser" name="my-browser">
```

Чуть поправил стиль кода